spell.ignore

```diff
@@ -2083,5 +2072,34 @@ zfgrepi
 zfile
 zgrep
 zgrepi
+zlib
+zstd
 Zuo
 zzz
+sizeunit
+refcnt
+mnonce
+getfd
+multipeerroundrobin
+groupname
+nodexx
+vq
+ioport
+yy
+KVER
+maxMemory
+valuenn
+fcntl
+tapname
+hgps
+slic
+tlbflush
+openfiles
+memoryBacking
+nosp
+zpci
+Devcontainer
+subuid
+subgid
+uids
+gids
```

Comment:
Just one question at `spell.ignore`: Do we need to obey the alphabetical order?
Long time ago, I heard from somebody: In this file,  we should obey the alphabetical order.

Thanks!

Reply:
This file includes 2 parts, one is `# The list below needs a second review`.

I hope we can order it, but https://github.com/ana/avocado-vt/blob/9ffc7e968bced5104d3ef28b88f1fbedd581fb4b/spell.ignore introduced too many words, and now the file is messy, so I have no idea how to maintain this file.

Reply:
And actually, I considered `codespell` instead of pylint. I think pylint is not focused on spell check.

Reply:
Hello @richtja, can you review this(`codespell` vs `typos` vs `pylint`) from your perspective against the whole avocado-framework?

Reply:
Hi @PaulYuuu and @nickzhq, IMO we should keep pylint for spell check. It works well across the avocado organization, and we already have it in avocado-vt requirements for static checks. About the alphabetical order, IMO it is not needed. In Avocado, we don't have it and it works well.